README.rst

```diff
@@ -55,10 +55,13 @@ Ticker and RPC connection information, currency.toml:
 ::
```

Comment:
This `README.rst` should be in the `plugin` folder. Create a new `README` for the entire repo with basic instructions on how to run everything.